gateway/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"context"
```

Comment:
can you organise a bit the imports like following in the diverse go files of this PR?

```
import (
	"context"
	"net/http"
	"os"

	"github.com/rs/zerolog"
	"github.com/sirupsen/logrus"
	"github.com/ekkinox/grpc-demo/proto/github.com/ekkinox/grpc-demo/proto"
	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
	"google.golang.org/grpc"
	"google.golang.org/grpc/credentials/insecure"
)
```

I did not always do it, but it help recognize go standard modules vs vendors